src/test/java/com/cdp/taf/stepdef/LoginStepDef.java

```diff
@@ -0,0 +1,24 @@
+package com.cdp.taf.stepdef;
+
+
+import com.cdp.taf.bo.LoginRegisterBO;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.testng.Assert.assertTrue;
+
+public class LoginStepDef extends BaseStepDef {
```

Comment:
I`m not sure do we need to extend BaseStepDef (which is actually hooks class) or not